packages/wdio-browser-runner/src/utils.ts

```diff
@@ -54,6 +54,28 @@ export function makeHeadless (options: BrowserRunnerOptions, caps: WebdriverIO.C
     return caps
 }
 
+export function disablePasswordManagerLeakDetection (caps: WebdriverIO.Capabilities): WebdriverIO.Capabilities {
```

Comment:
This is not needed because we start browser through `@wdio/utils` in the same way for browser component tests as for e2e tests. We only have the `makeHeadless` feature for browser component tests that enables headless testing automatically when run in CI

Reply:
Great feedback, will revert it later tonight 👍